internal/controllers/nmc_reconciler.go

```diff
@@ -993,8 +1014,32 @@ func (p *podManagerImpl) baseWorkerPod(
 			MountPath: "/usr/lib/modules",
 			ReadOnly:  true,
 		},
+		{
+			Name:      volNameTmp,
+			MountPath: sharedFilesDir,
+			ReadOnly:  true,
+		},
 	}
 
+	const initContainerArgTmpl = `
```

Comment:
i think that suing templates in this case is an overkill: we have a very short string, which can easily be produced by Sprintf oranother simple option

Reply:
I think that is the only way I found in order to have a clean arg with different commands in the `pod.initContainers[].arg` which isn't super long with additional `\n` in it.

Reply:
Changed to `fmt.Sprintf`.